### DISCUSSION TITLE
🐛 Bug Report: Unclear entity field labels - created at, updated at

### DISCUSSION THREAD

**Discussion by MoFoGo** [amplication/amplication#2291] [General]
### What happened?

In the Entity window, the fields Created at and Updated at would be easier to understand if they were renamed to:
Date Created
Date  Updated 

### What you expected to happen

Created at - could be understood as the location it is created, not the time. 

### How to reproduce

GUI Lable

**Comment by mshidlov** [comment]
Hey @MoFoGo thank you for submitting that issue.

`updated at` and `created at` are naming conventions for timestamp fields: 

created_at, createdAt, CREATED_AT // Time when the record was created
updated_at, updatedAt, UPDATED_AT // Time when the record was last modified
started_at, startedAt, STARTED_AT // Some other record time

I'm converting this issue for a discussion in case more people find this unclear and will have a different naming convention suggestion.

**Comment by yuval-hazaz** [comment]
I agree with @mshidlov 
see this example from Prisma https://www.prisma.io/docs/concepts/components/prisma-schema
